tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/encoder/PlainEncoder.java

```diff
@@ -66,46 +54,29 @@ public void encode(boolean value, ByteArrayOutputStream out) {
 
   @Override
   public void encode(short value, ByteArrayOutputStream out) {
-    if (this.endianType == EndianType.LITTLE_ENDIAN) {
-      out.write(value & 0xFF);
-      out.write((value >> 8) & 0xFF);
-    } else if (this.endianType == EndianType.BIG_ENDIAN) {
-      out.write((value >> 8) & 0xFF);
-      out.write(value & 0xFF);
-    }
+    out.write((value >> 8) & 0xFF);
+    out.write(value & 0xFF);
   }
 
   @Override
   public void encode(int value, ByteArrayOutputStream out) {
-    if (this.endianType == EndianType.LITTLE_ENDIAN) {
-      out.write(value & 0xFF);
-      out.write((value >> 8) & 0xFF);
-      out.write((value >> 16) & 0xFF);
-      out.write((value >> 24) & 0xFF);
-    } else if (this.endianType == EndianType.BIG_ENDIAN) {
-      out.write((value >> 24) & 0xFF);
-      out.write((value >> 16) & 0xFF);
-      out.write((value >> 8) & 0xFF);
-      out.write(value & 0xFF);
-    }
+    ReadWriteForEncodingUtils.writeVarInt(value, out);
   }
 
   @Override
   public void encode(long value, ByteArrayOutputStream out) {
-    if (this.endianType == EndianType.LITTLE_ENDIAN) {
-      for (int i = 0; i < 8; i++) {
-        out.write((byte) (((value) >> (i * 8)) & 0xFF));
-      }
-    } else if (this.endianType == EndianType.BIG_ENDIAN) {
-      for (int i = 7; i >= 0; i--) {
-        out.write((byte) (((value) >> (i * 8)) & 0xFF));
-      }
+    for (int i = 7; i >= 0; i--) {
+      out.write((byte) (((value) >> (i * 8)) & 0xFF));
     }
   }
 
   @Override
   public void encode(float value, ByteArrayOutputStream out) {
-    encode(Float.floatToIntBits(value), out);
+    int floatInt = Float.floatToIntBits(value);
```

Comment:
Previous version use encode(int value , ByteArrayOutputStream out) which in the previous version is not encode it as varInt. However, in the current version encode(int value , ByteArrayOutputStream out) encode the integer as a varInt, so the encode(float value, ByteArrayOutputStream out) can no longer call that method, because we don't want our float value become a varInt.